packages/next-ui/ActionCard/ActionCard.tsx

```diff
@@ -265,7 +266,7 @@ export function ActionCard(props: ActionCardProps) {
           width: '100%',
           justifyContent: 'space-between',
           alignContent: 'stretch',
-          alignItems: 'center',
+          alignItems: 'stretch',
         }}
```

Comment:
@Jessevdpoel Heb je dit (en justifyContent: 'space-between' hieronder op regel 292) goed getest bij de bestaande implementaties van ActionCard?

Reply:
This does not affect existing implementations

Reply:
Do all shipping and payment steps still look as intended?

Reply:
Moved styling from ActionCard to CartItemActionCard